### ISSUE TITLE
Active maintainer needed

### ISSUE THREAD

**Issue by dlo** [lionheart/django-pyodbc#158]
Hi everyone who watches django-pyodbc issues! 👋 

Development has been in quasi maintenance mode for a while. @RossRogers and I have both been sifting through the open issues lately, but both of us have been slammed with work, so progress has been limited. Furthermore, I haven't used MSSQL on a Django project in a while, and ideally whoever takes lead is actively using it. TBH, I probably should have posted this years ago...but life happened. 😅

There are a few open items that need addressing, like Django 2.0 bugs (#156), Python 3 issues (#47), tests failing (#9 #66 #102), which are all pretty high priority items that none of the current contributors have had time to address.

If you're interested in taking over, please just submit a PR for any of the open issues from the issues list with a short bio. If you need ideas, #110 and #47 are the highest priority, based on #'s of 👍's. Getting tests running on a CI service is also super high on the list (not sure how we do that with MSSQL? maybe we could get Microsoft/Azure to lend a server for testing purposes?).

BTW, if you've already contributed to the repo and just didn't realize there was an open spot for a maintainer, now's your chance to step up! Just comment below.

Please share within your networks, too, if you know anyone who might have time to take this on! Thanks in advance. :)

#### Responsibilities:

* Pushing new versions to PyPi.
* Updating documentation.
* Reviewing PRs.
* Responding to issues.

**Comment by ofalk** [comment]
Indeed, I've submitted a PR, some time ago (without looking it up, I wasn't even able to recall what it was). Anyway, at that point in time I've started a Django project that had to connect to some (read only) MSSSQL DB. However, the project was stopped and therefore I'm not using django-pyodbc any longer.

I other words, although I don't want this to die (in case django-pyodbc-azure is not a valid alternative/replacement), I'm probably the wrong person to maintain it.

**Comment by beruic** [comment]
I use django-pyodbc-azure for MSSQL support in my projects, and so far it works. Perhaps @michiya is interested in merging the two projects?

**Comment by bloodywing** [comment]
I was about to use django-pyodbc in the past because of IBM DB2 and they have their own DB driver for django (Which is a mess btw too :)). I don't have the free resources to maintain django-pyodbc nor the insight how the driver actually works and what needs to be done.

**Comment by benmehlman** [comment]
For a long time now I have been running production on my fork of django-pyodbc with two patches I worked out when I switched to django 1.10.. which I am still on.  One of those patches were finally merged into the project, the other wasn't.. but it works for me.   I understand that eventually this will break when I upgrade django, and I will be stuck and probably have to try azure...

Our use of MSSQL here is limited to one part of one application.. we use postgres everywhere else.. and I'm not really all that familiar with how the database backends work.. so I'm really not a good candidate for maintainer.

**Comment by RossRogers** [comment]
@bloodywing, that's why I'm on django-pyodbc too. I need the IBM DB2 support and I don't think django-pyodbc-azure supports one of the antique installations of MS SQL Server that I have to connect to. 
 Ultimately, if you're using django, odds are you're oriented more towards open source databases for your green field work.
